pkg/kyma/cmd/test/delete/cmd.go

```diff
@@ -0,0 +1,70 @@
+package del
+
+import (
+	"fmt"
+
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/kyma-project/cli/pkg/api/octopus"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test"
+	"github.com/kyma-project/cli/pkg/kyma/core"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *options
+	core.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: core.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "delete <test-suite name>",
+		Short:   "Delete tests on a running Kyma cluster",
```

Comment:
```suggestion
		Short:   "Delete tests available for a provisioned Kyma cluster",
```

I was just wondering about the word "running" - maybe `provisioned` would be good as well? What do you think? I changed it but we may revert it to the original.